### PR TITLE
✨ Anime violation command

### PR DESCRIPTION
**Pull request by dftd**
The weebs are taking over devcord! We shall stop them once for all!

A cool little command that sends an "no anime violation" image filled with the current time, place (channel), issued to, issued by, and an optional penalty.

Some doubts:
- where should the template image (anime.png) be stored?
- is my regex for channel name/penalty alright? I wanted to filter out everything non-alphanumeric
- shall the command be limited only to moderators?

✨ Inspired and mentored by Justas. 

image

### CODE REVIEW

File: commands/util/anime.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    client,
+  }): Promise<void> {
+    const [userId, ...restArgs] = args
+
+    if (!userId) return
+
+    const date = new Date().toLocaleString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit', hour: '2-digit', minute: '2-digit' })
+
+    const channel = await client.channels.fetch(message.channel.id)
+    const channelName = '#' + (channel as TextChannel).name.replace(/[^a-zA-Z0-9\-]/g, '')
```
Comment (austinmccalley) [inline commands/util/anime.ts:23]: Bad TypeScript practice please fix

File: commands/util/anime.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        canvasCtx.fillText(channelName, 150, 470);
+        canvasCtx.fillText(issuedTo.username, 230, 515);
+        canvasCtx.fillText(issuedBy.username, 230, 560);
+        canvasCtx.fillText(penalty, 190, 600);
+
+        const attachment = new MessageAttachment(canvas.toBuffer(), 'attachment.png')
+
+        await message.channel.send(attachment)
+    } catch (error) {
+      console.log(error)
```
Comment (austinmccalley) [inline commands/util/anime.ts:50]: On error please react to message with an X

**Review by austinmccalley — CHANGES_REQUESTED** [review]
Please make the following changes before we consider a merge

**Comment by dftd** [comment]
Sure, please check is it alright.

**Pull request merged**